xml/security-tls_mysql.xml

```diff
@@ -102,7 +102,7 @@ mysql_gcomms_bind_tls: False</screen>
     <para>
      Uncomment the appropriate line so the file looks like this:
     </para>
-<screen># TLS disabled for cluster
+<screen># TLS enabled for cluster
```

Comment:
shouldn't `# TLS enabled for cluster` be uncommented in order to enable TLS?

Reply:
Definitely not. It is correct as-is.